src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss

```diff
@@ -1,3 +1,29 @@
 .sortable-header {
   cursor: pointer;
 }
+
+#copy-question-modal {
+  table-layout: auto;
+  margin-bottom: 50px;
+}
+
+.modal-footer {
+  position: fixed;
+  bottom: 0;
+  height: 50px;
+  width: 69.8%;
+  justify-content: right;
+  background-color: white;
+}
+
+td {
+  text-align: center;
+  vertical-align: middle;
+}
```

Comment:
I am not entirely sure the central alignment improves the readability. I will leave this to other reviewers' opinions.

**Original:**
<img width="918" alt="Screenshot 2021-02-19 at 13 34 55" src="https://user-images.githubusercontent.com/32880438/108462448-976cbe80-72b7-11eb-83fe-226328410b74.png">

**After:**
<img width="922" alt="Screenshot 2021-02-19 at 13 35 41" src="https://user-images.githubusercontent.com/32880438/108462483-a2bfea00-72b7-11eb-920d-3c67ffc1a7b1.png">

Reply:
Speaking for myself: I'd prefer the original look. That allows reading in a more natural manner.

Reply:
For UI decisions such as this one, best to check with @damithc

Reply:
Previously we have standardized that such tables with text should be left-aligned (i.e. original)

Reply:
I can make those changes and have them in tomorrow if need be. That was my bad, I forgot that the earlier demo questions tended to have shorter questions that appeared to have centered check boxes and I did not want to break that alignment in adding scaling. If the original left-aligned is the standard, I can go back and make sure everything fits that standard.

Reply:
@t-cheepeng was involved in standardizing this kind of UI issues. @t-cheepeng do we have a document of the decisions taken? If not, we should.

Reply:
@damithc Yes, we do have a document for this. But it was not made made public/posted on github

Reply:
> @damithc Yes, we do have a document for this. But it was not made made public/posted on github

I see. Let's add it to our docs. At least a public link to the Google doc.